README.md

```diff
@@ -341,6 +342,30 @@ a public repository, we do not use this functionality here._
 
 ### Miscellaneous
 
+##### Logging
+
+The `logging` package is used to track events after running the project. The main logged events (levels) in Secret Santa are: errors, warnings, and participants info. A log level is set as an environment variable, e.g.:
+```bash
+os.environ["level"] = "ERROR"
+```
+
+
+All logs activities are collected into a log file that is initiated at the beginning of the code:
+```bash
+logging.basicConfig(filename = 'file_name')
```

Comment:
maybe you would like to update this command (?)

Reply:
command inputs have been updated